tomviz/RecentFilesMenu.cxx

```diff
@@ -208,6 +216,17 @@ void RecentFilesMenu::dataSourceTriggered()
   QAction* actn = qobject_cast<QAction*>(this->sender());
   Q_ASSERT(actn);
 
+  QFileInfo checkFile(actn->iconText());
+  if (! checkFile.exists() )
```

Comment:
Let's stick with no space after binary operators and no additional space before the closing brace, i.e.

if (!checkFile.exists())